src/core/utils/helpers/general.ts

```diff
@@ -176,6 +176,14 @@ export const sortByLoanDate = (list: LoanType[]) => {
   );
 };
 
+export const sortByReservationDate = (list: ReservationType[]) => {
+  return list.sort(
+    (objA, objB) =>
+      new Date(objA.dateOfReservation || new Date()).getTime() -
```

Comment:
When are we using dayjs and when not? Just curious.

Reply:
I'm pretty sure this is a copy paste from another function, where I tried sorting using dayjs and for some reason it did not work. I dont quite remember, but I can try again..?